cocos/2d/components/rich-text.ts

```diff
@@ -508,7 +508,7 @@ export class RichText extends Component {
     protected _labelHeight = 0;
     protected _layoutDirty = true;
     protected _lineOffsetX = 0;
-    protected _updateRichTextStatus: () => void;
+    protected declare _updateRichTextStatus: () => void;
```

Comment:
Can also use declare for `_userDefinedFont ` ?

Reply:
We couldn't, because it's decorated by `@serailzable`.
```ts
@serializable
    protected _userDefinedFont: TTFFont | null = null;
```

But yes, I think there is not need to assign it to null in `constructor`. I will update it.